src/backend/marsha/core/serializers/video.py

```diff
@@ -318,6 +318,19 @@ class PairingChallengeSerializer(serializers.Serializer):
     secret = serializers.CharField(min_length=6, max_length=6)
 
 
+class LiveRegistrationDisplayUsernameSerializer(serializers.ModelSerializer):
+    """Serializer for liveRegistration models and display_name."""
+
+    class Meta:
+        model = LiveRegistration
+        fields = ("anonymous_id", "display_name", "username")
+        read_only_fields = ("username",)
+        extra_kwargs = {
+            "anonymous_id": {"allow_null": False, "required": True},
```

Comment:
I don't know if the `anonymous_id` should be required. If I'm not an anonymous, I don't need this anonymous_id and you don't use it in the API

Reply:
I wondered as well, but the front wouldn't know as it's the back that checks it, wdyt ?

Reply:
With this behaviour, every user, anonymous or not, will have an anonymous_id stored in the front application. We can in a first time have one and if we detect an unwanted side effect we will remove it. So keep it for now.